conf/susi/susi_cognition_000.json

```diff
@@ -712,6 +712,16 @@
 			"actions":[	{"type":"answer", "select":"random", "phrases":[
 							"I like to chat with people!"
 						]}]
-		}
+		},
+		{
+			"keys"   :["github"],
+			"score"  :3000,
+			"example": "Susi, View the Github profile of mariobehling",
+			"phrases":[ {"type":"pattern", "expression":"* github * of *"}],
```

Comment:
Yes @sudheesh001 . This was just a specific pattern, we can have more too. Shall add 'em up.